src/cloud/docker/docker-installation.md

```diff
@@ -0,0 +1,146 @@
+---
+group: cloud-guide
+title: Install prerequisites
+functional_areas:
+  - Cloud
+  - Configuration
+---
+
+{{site.data.var.mcd-prod}} is one of the {{site.data.var.csuite}} packages designed to deploy and manage local {{site.data.var.ee}} Docker environments for both Cloud and on-premises projects.
+
+If you are using {{ site.data.var.ee }} on Cloud, you most likely have the {{ site.data.var.mcd-package }} installed together with the {{site.data.var.ct-repo}} package.
+
+On-premises installation requires the stand-alone {{ site.data.var.mcd-package }} available in the [Magento Cloud Docker GitHub repository](https://github.com/magento/magento-cloud-docker).
+
+## Initialize the Magento project
+
+{{ site.data.var.mcd-prod }} supports {{site.data.var.ee}} installation for both Cloud and on-premises projects. The initialization steps are different for each project type.
+
+### Magento Commerce Cloud projects
+
+{:.procedure}
+
+1. Download a Magento application template from the [Magento Cloud repository][cloud-repo]. Be careful to select the branch that corresponds with the Magento version.
+
+1. You can also run next command to clone the latest template:
+
+   ```bash
+   git clone https://github.com/magento/magento-cloud.git <install-directory-name>
+   ```
+
+1. Change to the new project directory.
+
+1. Add your [Magento access credentials][magento-creds] to the `auth.json` file.
+
+### Magento Commerce projects
+
+When you initialize a {{site.data.var.ee}} project for Docker development, you must create a [.magento.docker.yml][unified configuration] configuration source file to create the Docker containers for the local environment.
+
+{:.procedure}
+
+1. Create a project using [Composer]({{site.baseurl}}/guides/v2.4/install-gde/composer.html)
+
+   ```bash
+    composer create-project --repository-url=https://repo.magento.com/ magento/project-enterprise-edition <install-directory-name>
+   ```
+
+   This command installs the latest Magento release. To install a specific version, specify the version number, for example `magento/project-enterprise-edition=2.3.5`.
+
+1. Change to the project directory.
+
+1. Add the {{ site.data.var.ct }} and {{ site.data.var.mcd-prod }} packages.
+
+   ```bash
+   composer require --no-update --dev magento/ece-tools magento/magento-cloud-docker
+   ```
+
+1. Create a default configuration file.
+
+   {{ site.data.var.mcd-prod }} requires a configuration source file to create the Docker containers for the local environment.
```

Comment:
This is a repeat of the intro paragraph. You could eliminate if you enhance the step...

```suggestion
1. Create the default configuration source file that builds the Docker containers for the local environment.
```